### PR TITLE
##150356328 fix paging issue in pagination

### PR DESCRIPTION
**Pull request by jmnyarega**
**What the PR does**
- Allow the user to move to the next page of the pagination object

**How should this be manually tested?**

- Login with valid credentials to receive an authentication token
- In a new tab set the request type to GET
- Enter the endpoint url in the Postman url field as: http://127.0.0.1:5000/bucketlists?limit=3&page=1

### CODE REVIEW

**Comment by nelmandela** [comment]
Great work.

File: tests_endpoints/base_testcase.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ class BaseTest(unittest.TestCase):
     def setUp(self):
         # setup test environment configuration
         app.config['TESTING'] = True
-        app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///bucket_test.db'
+        app.config.from_object('config.TestingConfig')
         self.client = app.test_client()
         self.u = UserStore()
```
Comment (HawiCaesar) [inline tests_endpoints/base_testcase.py]: I think self.u can be a little more descriptive.

File: tests_endpoints/test_bucketitems.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def test_get_with_query_bucketlistItem(self):
             '/bucketlists/', data=json.dumps(self.bucket), headers=self.set_header())
         self.client.post('/bucketlist/1/items/',
                          data=json.dumps(self.item), headers=self.set_header())
-        rv = self.client.get('/bucketlist/1/items/?q=Swim',
+        rv = self.client.get('/bucketlist/1/items/?limit=1&q=S',
                              headers=self.set_header())
         self.assertEqual(json.loads(rv.data.decode()),
```
Comment (HawiCaesar) [inline tests_endpoints/test_bucketitems.py]: self.assertEqual still works ok here though is mostly used for numbers or counts. self.assertIn is would more appropriate. I also used self.assertEqual. On further review I saw self.assertIn is better

File: tests_endpoints/base_testcase.py (hunk 1 above)
Comment (jmnyarega) [inline tests_endpoints/base_testcase.py]: Thanks @HawiCaesar  i will fix that

File: tests_endpoints/test_bucketitems.py (hunk 2 above)
Comment (jmnyarega) [inline tests_endpoints/test_bucketitems.py]: @HawiCaesar i will look into that.

**Pull request merged**